canonical-web-and-design/ubuntu.com
# 1292 remove openstack partners

## Done

- Removed /openstack/partners page and associated links
- Redirected /openstack/partners to /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/openstack/partners](http://0.0.0.0:8001/openstack/partners)
- See that you have been redirected to /openstack, and that "Partners" is not visible in the secondary navigation


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1292
